include/ignition/gazebo/detail/EntityComponentManager.hh

```diff
@@ -385,7 +385,8 @@ void EntityComponentManager::Each(typename identity<std::function<
   // function.
   for (const Entity entity : view->Entities())
   {
-    if (!std::apply(_f, view->EntityComponentConstData(entity)))
+    if (!std::apply(_f,
+          view->template EntityComponentConstData<ComponentTypeTs...>(entity)))
```

Comment:
I'm not well versed at knowing when to use the `template` keyword...I usually do what the compiler says. In this case, the keyword was necessary because of the `auto` when `view` was declared. If I change the type to `detail::View *`, the keyword is not necessary anymore.